packages/next/server/config.ts

```diff
@@ -643,7 +652,7 @@ export default async function loadConfig(
     }
   }
 
-  const completeConfig = defaultConfig as NextConfigComplete
+  const completeConfig = getDefaultConfig() as NextConfigComplete
```

Comment:
```suggestion
  const completeConfig = defaultConfig as NextConfigComplete
```

Reply:
`defaultConfig` is instantiated in `assignDefaults` and this line is in `loadConfig`. 

Moving it to the static scope would somewhat defeat the purpose of the factory function as well, as that opens the ability to mutate within this module scope.

Reply:
Let's pass it into `assignDefaults()` then?